src/seedu/addressbook/AddressBook.java

```diff
@@ -548,6 +553,23 @@ private static String executeListAllPersonsInAddressBook() {
         showToUser(toBeDisplayed);
         return getMessageForPersonsDisplayedSummary(toBeDisplayed);
     }
+    
+    /**
+     * Displays all persons in the address book to the user; sorted alphabetically
+     */
+    private static String executeSortAllPersonsInAddressBook() {
+    	ArrayList<String[]> toBeDisplayed = getAllPersonsInAddressBook();
+    	Collections.sort(toBeDisplayed, new Comparator<String[]>(){
+    		@Override
```

Comment:
Do take note of the coding standard. 4 space is to be used instead of tabs. You can easily configure the IDE to enter the correct indentation.